docs/manuals/gui/viselements/index.md

```diff
@@ -280,11 +278,11 @@ can use it in your CSS selectors. You can look at the [Styling](../styling/index
 section for more information.
 
 !!! note
-    This identifier is also sent to the *on_action* callback if this visual
+    This identifier is also sent to the `on_action` callback if this visual
```

Comment:
() ?

Reply:
Same comment as above: this is the callback notion from Taipy GUI's stand point.